src/utils/testing.tsx

```diff
@@ -177,11 +193,21 @@ export function unloadCSS(path) {
 // If a component requires certain props or other conditions in order
 // to render without errors, you can pass a `toRender` function that
 // returns an element ready to be rendered.
-export function behavesAsComponent(Component, systemPropArray, toRender = null, options) {
-  if (typeof toRender === 'object' && !options) {
-    options = toRender
-    toRender = null
-  }
+
+
+interface IOptions {
+  skipAs?: boolean
+  skipSx?: boolean
+}
+
+interface IBehavesAsComponent {
+  Component: React.FunctionComponent<any>,
+  systemPropArray: any[],
+  toRender?: () => React.ReactElement,
+  options?: IOptions
+}
+
+export function behavesAsComponent({Component, systemPropArray, toRender, options}: IBehavesAsComponent) {
```

Comment:
We don't usually prefix our interfaces with `I`:
```suggestion
interface Options {
  skipAs?: boolean
  skipSx?: boolean
}

interface BehavesAsComponent {
  Component: React.FunctionComponent<any>,
  systemPropArray: any[],
  toRender?: () => React.ReactElement,
  options?: Options
}

export function behavesAsComponent({Component, systemPropArray, toRender, options}: BehavesAsComponent) {
```